src/main/java/com/transportcompany/transportcompany/repositories/UserRepository.java

```diff
@@ -5,9 +5,13 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
     static User save(UserDTO userToSave) {
         return null;
     }
+
+    List<UserDTO> getUsers();
```

Comment:
Also here, as stated above, to return a simple list of users you don't need to declare a method for that. We can use what Spring Data JPA provides.
Please don't forget that we should return from the database entities, not the DTOs. If we for some reason we need a custom method in the repository layer, we would have:

```
List<Entity_You_Need> methodName();
or
List<User> methodName();
```

Reply:
Done